roles/jenkins_builder/tasks/jenkins_credential.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Placing ssh public key for Jenkins_int_rht
+  authorized_key:
+    user: jenkins
```

Comment:
@mscherer Wouldn't this fail because we don't have the ansible master key in the `jenkins` user?

Reply:
No, it just edit a different fail as root if we give a "user" parameters.